index.rst

```diff
@@ -10,20 +10,849 @@
+
+~60 seconds
+^^^^^^^^^^^
+This timescale is useful when examining trending or slowly varying effects, particularly for metrics like image quality or transparency.
+It is a timescale where people are closely watching, but not necessarily immediately reacting.
+The addition of this category was to provide flexibility in implementation as it may be such that the prioritization of metrics can be performed which may provide a useful free parameter during the implementation phase.
+However, it is imperative that the Rapid Analysis framework be able to keep up with the rate of images being acquired; where that rate is governed by the survey strategy visit duration (`FAFF-REQ-0051`_).
+In the case of taking two 15 second snaps, it is expected that the analysis would be done on the combined images.
+
+Again, the data products for the 60 second timescales are described in the Outputs section of the `Rapid Analysis Use-case <https://confluence.lsstcorp.org/display/LSSTCOM/Rapid+Analysis+Use-Case>`_.
+
+
+12-24 hours
+^^^^^^^^^^^
+This timescale is important for more general commissioning activities and performance assessment that could impact observations taken in the next or subsequent nights.
+Over this timescale, a full SFP pipeline needs to be run (`FAFF-REQ-0052`_).
+This must include the additional values that are calculated in the Rapid Analysis Framework, which will need to be added to the SFP pipeline.
+Re-calculation of these values enables a more detailed and higher-confidence data quality evaluation to be performed, including correlation with telemetry, environmental conditions, and previous conditions and/or observations.
+It also allows the teams to begin determining which subsets of data should be used to construct coadds/templates, begin science verification analyses, and ultimately maximize the number of human brain cycles looking at the data.
+It is fully expected that this dataset will be superseded by a subsequent DRP campaign to enforce that all the data is processed in a homogeneous way with best performing configuration of the science pipelines.
+
+It is not required that the full SFP processing be done in Chile, in fact, it is *preferable* to perform this processing at the USDF as many of the science verification tasks are planned to be performed there as well.
+It also ensures that a minimum number of users are connecting to Chile to perform their analyses.
+This is especially important if connections are required to the summit instance.
+
+Lastly, the results of this analysis do not need to be forwarded back to the summit control system.
+
+Potential Paths for Implementation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Rapid Analysis payload relies heavily on SFP, and therefore is a compatible payload with both the DRP and the Alert Production Pipelines.
```

Comment:
Consider replacing "payload" with "data products"...

> The Rapid Analysis data products relies heavily on SFP, and therefore is compatible with both the DRP and the Alert Production Pipelines.

Reply:
this was addressed above... where payload is the pipeline..